ui/statusbar/ui.go

```diff
@@ -86,7 +89,9 @@ func (model Model) View() string {
 		status = neutralStatusStyle.Render("Idle")
 
 	case lib.ERROR:
-		status = errorStatusStyle.Render("Error")
+		statusMsg := "Error"
+		status = errorStatusStyle.Render(statusMsg)
+		shortMessage = truncateMessage(shortMessage, halfWidth, len(statusMsg)*2)
```

Comment:
I see your reason for this. It's because of the escape characters affecting the width. But I think we can use `lipgloss.Size()` to get the actual string width.

It'd be nice if we refactored this back to the old code structure and then use `lipgloss.Size()` to calculate the width.

Reply:
Nice! I didn't know about this function ... let's change it.